viral/offline_sequence_detection.py

```
@@ -0,0 +1,567 @@
+    #         for x in np.linspace(grosmark_config.start, grosmark_config.end, 5).astype(
+    #             int
+    #         )
+    #     ],
+    # )
+    plt.tight_layout()
+    plt.savefig(f"plots/pse_events/{session.mouse_name}_{session.date}_event{idx}.png")
+
+
+def main() -> None:
+    """
+    Offline PSEs were detected by convolving each PC’s (as assessed during that day’s run) offline immobility firing rate vector
```

Ruff (0.14.1)

Docstring contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF002)